client/common/cmdline.h

```diff
@@ -224,6 +224,7 @@ static const COMMAND_LINE_ARGUMENT_A global_cmd_args[] = {
 	{ "kbd-unicode", COMMAND_LINE_VALUE_FLAG, "", NULL, NULL, -1, NULL,
 	  "Send unicode symbols, e.g. use the local keyboard map. ATTENTION: Does not work with every "
 	  "RDP server!" },
+	{ "kdc-url", COMMAND_LINE_VALUE_REQUIRED, "<url>", NULL, NULL, -1, NULL, "KDC server URL" },
```

Comment:
I think this should be part of the `/kerberos` options?

Reply:
@awakecoding sorry I've changed my mind about this option, I agree with @akallabeth, that would be nice to add this to `/kerberos`, and much simpler as the treatment already does all the checks (then it's just a matter of adding an entry in the array of options and set the target settings id).

We do the split first on coma so that `/kerberos:kdc-url:udp://IT-HELP-DC.ad.it-help.ninja:88,lifetime:2d,cache:/tmp/mycache` would work perfectly without the need for quoting the URL.

Could you also update the various help messages, so that we have an extensive documentation of all options

Reply:
I've moved the kdc-url option as a kerberos suboption instead